src/main/java/com/github/msemys/esjc/EventStoreBuilder.java

```diff
@@ -452,6 +453,18 @@ public EventStoreBuilder failOnNoServerResponse(boolean failOnNoServerResponse)
         return this;
     }
 
+    /**
+     * Sets whether or not to disconnect the client on detecting a channel error. By default, it is enabled and
+     * client disconnects immediately. If it is disabled the client tries to reconnect according to {@link #maxReconnections(int)}.
+     *
+     * @param disconnectOnTcpChannelError {@code true} to disconnect or {@code false} to try to reconnect.
+     * @return the builder reference
+     */
+    public EventStoreBuilder disconnectOnTcpChannelError(boolean disconnectOnTcpChannelError) {
```

Comment:
could you please update the javadocs

Reply:
Good catch, sorry I missed this one :see_no_evil: